src/main/java/io/xpring/xrpl/Utils.java

```diff
@@ -214,4 +215,14 @@ public static String xrpToDrops(String xrp) throws XrpException {
             .toBigInteger()
             .toString(10);
   }
+
+  /**
+   * Check if the XrplNetwork is a test network.
+   *
+   * @param xrplNetwork - The {@link XrplNetwork} to check.
+   * @return A boolean representing whether or not the {@link XrplNetwork} is a test network.
+   */
+  public static boolean isTestNetwork(XrplNetwork xrplNetwork) {
+    return xrplNetwork != XrplNetwork.MAIN;
```

Comment:
i chose to add `isTestNetwork` instead of `isMainnet` because we tend to check whether or not something is a test, not whether or not something "isn't" mainnet, if that makes sense. let me know if you think i should flip it...also if you think the language can be improved.

i also went for a `!= XrplNetwork.MAIN` check to be more defensive